src/elexmodel/handlers/data/Estimandizer.py

```diff
@@ -14,22 +14,14 @@ class Estimandizer:
     Estimandizer. Generate estimands explicitly.
     """
 
-    def check_and_create_estimands(self, data_df, estimands, historical, current_data=False):
+    def check_and_create_estimands(self, data_df, estimands, historical):
         columns_to_return = []
-
         for estimand in estimands:
             results_col = f"{RESULTS_PREFIX}{estimand}"
-            baseline_col = f"{BASELINE_PREFIX}{estimand}"
-            target_col = results_col if current_data else baseline_col
-
-            if target_col not in data_df.columns:
-                if estimand in data_df.columns:
-                    data_df[target_col] = data_df[estimand].copy()
-                else:
-                    # will raise a KeyError if a function with the same name as `estimand` doesn't exist
-                    data_df = globals()[estimand](data_df)
-                if target_col == baseline_col:
-                    data_df[results_col] = data_df[baseline_col].copy()
+
+            if results_col not in data_df.columns:
+                # will raise a KeyError if a function with the same name as `estimand` doesn't exist
+                data_df = globals()[estimand](data_df, RESULTS_PREFIX)
 
             if historical:
```

Comment:
why is this part necessary?

Reply:
I migrated code from `MockLiveDataHandler` in the previous PR that did this.  I didn't want anything to break.  Hmmmm 🤔

Reply:
So I think it makes sense in the `MockLiveData` case when historical is true (since we are overwriting those results anyway), but it doesn't make sense in the `PreprocessedData` if we call that function on line 61 (since we need the results for the historical run). I think it might be worth just getting rid  of the `nan` generally? We don't need those results in the first case, but they aren't actually a problem, I think?

Reply:
I see, hmmmm...ok, I'll take it out and see what breaks 😄